cmd/kubeops/internal/handler/dashboardcompat.go

```diff
@@ -0,0 +1,121 @@
+// The Dashboard doesn't understand the Helm 3 release format.
+// This file is a compatibility layer that translates Helm 3 releases to a Helm 2-similar format suitable for the Dashboard.
+
+package handler
+
+import (
+	"strings"
+
+	"github.com/golang/protobuf/ptypes/any"
+	"gopkg.in/yaml.v2"
+	h3chart "helm.sh/helm/v3/pkg/chart"
+	h3 "helm.sh/helm/v3/pkg/release"
+	h2chart "k8s.io/helm/pkg/proto/hapi/chart"
+	h2 "k8s.io/helm/pkg/proto/hapi/release"
+)
+
+// generatedYamlHeader is prepended to YAML generated from the internal map[string]interface{} representation.
+const generatedYamlHeader = "# Not original YAML! Generated from parsed representation."
+
+type dashboardCompatibleRelease struct {
+	Name      string                    `json:"name,omitempty"`
+	Info      h2.Info                   `json:"info,omitempty"`
+	Chart     dashboardCompatibleChart  `json:"chart,omitempty"`
+	Config    dashboardCompatibleConfig `json:"config,omitempty"`
+	Manifest  string                    `json:"manifest,omitempty"`
+	Version   int                       `json:"version,omitempty"`
+	Namespace string                    `json:"namespace,omitempty"`
+}
+
+type dashboardCompatibleChart struct {
```

Comment:
Same here with https://github.com/helm/helm/blob/release-2.16/pkg/proto/hapi/chart/chart.pb.go#L24 (although I see you're using `h3chart.Metadata` below - are they the same?)

Reply:
No, `h2chart.Metadata` and `h3chart.Metadata` are different (and not isomorphic).